backend/runtime/Cargo.toml

```diff
@@ -111,6 +111,8 @@ std = [
 	"pallet-ajuna-tournament/std",
 	"pallet-sage/std",
 	"sage-api/std",
+	"game-battle-mogs/std",
+	"game-casino-jam/std",
```

Comment:
This makes me realize, where is zepter? ;)

Reply:
I'll add a zepter step then :)